integration/tests/cook/test_basic.py

```diff
@@ -1119,18 +1119,22 @@ def test_hostname_equals_job_constraint(self):
             host_to_job_uuid = {}
             for hostname in hosts:
                 constraints = [["HOSTNAME", "EQUALS", hostname]]
-                job_uuid, resp = util.submit_job(self.cook_url, constraints=constraints)
+                job_uuid, resp = util.submit_job(self.cook_url, constraints=constraints,
+                                                 name='test_hostname_equals_job_constraint')
```

Comment:
1. Can we use the [`id()` method](https://docs.python.org/3/library/unittest.html#unittest.TestCase.id) to generate this name?

2. Should we make this strategy the default name (when one is not explicitly provided) for all our jobs in the tests?

Reply:
I switched this to use the `current_name()` function that is already in use by some of the CLI tests. As for number 2, that's a little trickier, because we generate the default fields in `util`, which doesn't have access to the `id()` method. I would vote to move that effort to a separate PR.

Reply:
Sounds good.